src/snowflake/connector/cursor.py

```diff
@@ -974,8 +974,7 @@ def execute(
                 )
             query = query1
 
-        if logger.getEffectiveLevel() <= logging.INFO:
-            logger.info("query: [%s]", self._format_query_for_log(query))
+        logger.debug("query: [%s]", self._format_query_for_log(query))
```

Comment:
How about log the original text without replacing the parameters? Then the log looks like server-side binding.

Reply:
sure